examples/jupyter-keras-tract/src/main.rs

```diff
@@ -0,0 +1,29 @@
+use tract_core::ndarray;
+use tract_core::prelude::*;
+use rand::*;
+//use tract_core::analyser::types::shapefact;
+//use tract_core::analyser::types::*;
+
+fn main() -> TractResult<()> {
+    // load the model
+    let mut model = tract_tensorflow::tensorflow().model_for_path("./my_model.pb")?;
+
+    // specify input type and shape
+    //model.set_input_fact(0, InferenceFact::dt_shape(f32::datum_type(), tvec!(_, 100)))?;
+
+    model.set_input_fact(0, InferenceFact::dt_shape(f32::datum_type(), tvec![10, 100]))?;
+    // Generate some input data for the model
```

Comment:
Well, one final thingy here, and that was the entire point: now that you have specified the input shape, you can call into_optimized() and get an even shorter running time :)